hydra-node/test/Hydra/OptionsSpec.hs

```diff
@@ -25,8 +25,14 @@ import Hydra.Options (
   toArgs,
   validateRunOptions,
  )
+import Options.Applicative (
+  ParserResult (..),
+  renderFailure,
+ )
```

Comment:
Should: Not undo the re-export, but add `renderFailure` to it.

As discussed yesterday, having a re-export of types and functions to use things returned by `Hydra.Options` in this case is helpful. It avoids an additional dependency on `optparse-applicative` on the call site. While I also not always do it out of lazyness, I acknowledge it as good practice which can also lead to much cleaner import lists.

Reply:
I’m not sure, especially since we really don’t use `renderFailure` anywhere in production code.
I can easily imagine some conversation like this, at some point:

> - let’s refactor this…
> - Oh but wait! We can’t, look, the module is re-exporting these types so we can’t just refactor as planned
> -  you’re right… let’s see why are we re-exporting this?
> - It has to be needed for something, of course, but I can’t find any use.
> - Maybe it’s used by external software? Do we export this as a library? If so we can’t just remove it, it would break the interface.
> - You’re right, let’s see but…
> - Yeah I agree, I don’t think we export that as a library…
> - But we do use them in the tests though.
> - Ok… let’s check if nobody is using this as a library in another external software first, I’ll post a message on slack. If so, then we’ll be able to patch the test and refactor.

Instrumenting production code for the sake of testing it always introduces some sort of noise. When you need it, you need it. But every time you can avoid noise you should.

But I understand that it’s important for you so I might miss something here, probably. Let’s move forward and go your way.

  I applied your suggested change.